src/services/todos.js

```diff
@@ -0,0 +1,24 @@
+import { checkError, client } from './client';
+
+export async function getTodos() {
+  const response = await client.from(`ToDoList`).select(`*`);
+  // console.log(checkError(response));
+  return checkError(response);
```

Comment:
Here we can add an order by function so that our data doesn't get rearranged after a completion.